package.json

```diff
@@ -31,8 +31,8 @@
     "pkg:aio": "yarn workspace @kaizen/components",
     "pkg:aio:test": "yarn pkg:aio test",
     "pkg:aio:compile": "yarn pkg:aio compile",
-    "ci:version": "yarn build && yarn changeset version",
-    "ci:publish": "yarn build && yarn changeset publish"
+    "ci:version": "yarn turbo prepublish && yarn changeset version",
+    "ci:publish": "yarn turbo prepublish && yarn changeset publish"
```

Comment:
Technically these could remain as `yarn build` lol. But not fussed if you don't change it.

Reply:
Eh, I like this one being explicit. Also won't need to run clean, making CI "faster" 😄